### PR TITLE
Circle activate workflows

### PR DESCRIPTION
**Pull request by k-dominik**

### CODE REVIEW

**Comment by m-novikov** [comment]
If we use `continuumio/miniconda3` image then to fix all conda init issues we would only need to set environment variable `BASH_ENV: /opt/conda/etc/profile.d/conda.sh` in yaml.

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,82 +1,137 @@
...
+    - save_cache:
+        key: v1-repo-{{ .Environment.CIRCLE_SHA1 }}
+        paths:
+        - ~/ilastik
+
+  configure_conda:
+    working_directory: ~/ilastik/ilastik
+    parallelism: 1
+    environment:
+      BASH_ENV: /opt/conda/etc/profile.d/conda.s
```
Comment (m-novikov) [inline .circleci/config.yml]: Typo in path should end with .sh

Comment (k-dominik) [reply .circleci/config.yml]: I've removed it completely - the miniconda3 container already seems to be correctly configured

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,82 +1,130 @@
...
         cd ${ILASTIK_ROOT}/ilastik-meta/ilastik &&
         mkdir test-results &&
         xvfb-run --server-args="-screen 0 1024x768x24" pytest --run-legacy-gui --junitxml=test-results/junit.xml
+    - store_test_results:
+        path: test-results
+    - save_cache:
+        name: store dependency cache
+        key: v1.4.8-dep-{{ .Branch }}-{{ epoch }}
+        paths:
+        - /opt/conda/pkg
```
Comment (m-novikov) [inline .circleci/config.yml]:
```suggestion
        - /opt/conda/pkgs
```

Comment (k-dominik) [reply .circleci/config.yml]: thx

**Comment by k-dominik** [comment]
3000 force-pushs later....

**Pull request merged**